Gardner.css

```diff
@@ -0,0 +1,386 @@
+}
+.welcome .weltext
+{
+    width: 50%;
+}
+.welcome .welcont
+{
+    width: 95%;
+    color: #949494;
+}
+.welbutton a
+{
+    display: inline-block;
+    background: #ffffff;
+    color: #709D61;
+    font-size: 16px;
+    padding: 12px 36px;
+    margin-top: 30px;
+    border-radius: 6px;
+    border: 2px solid #709D61;
+    transition: all 0.3s ease;
+    font-weight: bold;
+}
+.welbutton a:hover
+{
+    background: #709D61;
+    color: #ffffff;
+}
+/* landscape soln styling */
+.landscape .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+    text-align: center;
+    margin-top: 70px;
+}
+.landscape .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   text-align: center;
+   margin-bottom: 60px;
+}
+.land-content
+{
+    margin-top: 10px;
+    display: flex;
+    width: 1200px;
+}
+.land
+{
+    font-weight: bold;
+    color: #3a3a3a;
+    width: 100%;
+    margin: auto;
+    font-size: 20px;
+    margin-top: 20px;
+}
+.land-cont
+{
+    width: 90%;
+    color: #949494;
+    margin: 0;
+    margin-top: 20px;
+    margin-bottom: 60px;
+}
+/* work styling */
+.work .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+    text-align: center;
+    margin-top: 70px;
+}
+.work .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   text-align: center;
+   margin-bottom: 60px;
+}
+.work .work-content
+{
+    text-align: center;
+    padding: 10px;
+    padding-right: 10px;
+}
+.testimonial .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+    text-align: center;
+    margin-top: 70px;
+}
+.testimonial .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   text-align: center;
+   margin-bottom: 60px;
+}
+.test-text
+{
+    float: right;
+    width: 55%;
+}
+.test-text p,h3
+{
+    width: 70%;
+    color: #949494;
+    margin-top: 10px;
+}
+.ceo
+{
+    margin-top: 20px;
+    font-weight: bold;
+    font-size: 20px;
+    margin-bottom: 0;
+}
+.news .max-width
+{
+    background-color: #E0E8E1;
+    height: 250px;
+    margin-top: 80px;
+    margin-bottom: 30px;
+}
+.news .max-width .offer
+{
+    color: #709D61;
+    font-size: 16px;
+    font-weight: normal;
+    align-items: center;
+    text-align: center;
+    padding-top: 30px;
+}
+.news .max-width .whatoffer
+{
+   color:#000000;
+   font-weight: bold;
+   font-size: 45px;
+   text-align: center;
+   margin-bottom: 30px;
+}
+.news .max-width input
+{
+    height: 60px;
+    width: 580px;
+    border-radius: 10px;
+    border-style: none;
+    font-size: 18px;
+    padding: 20px;
+}
+.textbox
+{
+    text-align: center;
+}
+.news .max-width button
+{
+    border-radius: 5px;
+    width: 140px;
+    border: 3px solid;
+    border-color: #709D61;
+    right: 480px;
+    top: 525%;
```

Comment:
Please redo the entire styling for button

Reply:
Okay